pkg/metadata/metadata.go

```diff
@@ -121,6 +124,11 @@ func ApplyRebootAnnotation(ctx context.Context, c client.Client, node core.Node)
 	return ApplyLabelsAndAnnotations(ctx, c, node, nil, map[string]string{RebootAnnotation: ""})
 }
 
+// ApplyUnhealthyAnnotation applies an annotation to the given Node communicating that the instance is unhealthy
+func ApplyUnhealthyAnnotation(ctx context.Context, c client.Client, node core.Node) error {
```

Comment:
I assume you will include this change in the commit where these method will actually be used. I know you mentioned this is a WIP PR but calling this out so that there is no extra work for you in the future.

Reply:
Correct, I prefixed the commit accordingly [fixup: [metadata] Introduce unhealthy annotation](https://github.com/openshift/windows-machine-config-operator/pull/1901/commits/3280d32ae3add909ac08b826d4e6daea2a4d7042)